### PR TITLE
Recipe for SIDPlayer.

### PR DESCRIPTION
**Pull request by arnavbhatt288**
Recipe for SIDPlayer. Keep in mind that it doesn't uses any makefile (even BeOS makefile after updating it didn't work) so I have to use configure to make new makefile for it.
GCI Student: arnavbhatt288

### CODE REVIEW

File: haiku-apps/sidplayer/sidplayer-4.4.recipe [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	cmd:gcc
+	cmd:make
+	cmd:autoreconf
+	cmd:aclocal
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure --omit-dirs binDir ./configure --bindir=$binDir
```
Comment (korli) [inline haiku-apps/sidplayer/sidplayer-4.4.recipe]: Isn't --bindir already the standard option used by runConfigure?

Comment (arnavbhatt288) [reply haiku-apps/sidplayer/sidplayer-4.4.recipe]: @korli I don't know.

Comment (korli) [reply haiku-apps/sidplayer/sidplayer-4.4.recipe]: Then just use: "runConfigure ./configure"

**Comment by arnavbhatt288** [comment]
Hello, I will make more recipes for Haiku but since I need to contribute for other project (ReactOS), it will take some time.
Hopefully I will make new PR soon :)

**Pull request merged**